impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java

```diff
@@ -529,9 +519,7 @@ protected boolean isFullyQualified(String href) {
                 c = href.charAt(2);
                 if (c == 't' || c == 'T') {
```

Comment:
I'd recommend cleaning this up by using `toLowerCase`

Reply:
I've been eying this one too, I'm not sure why this was originally done this way.  I'm going to follow up with another PR for things like this.